app/models/claim/geckoboard_dataset.rb

```diff
@@ -39,6 +40,7 @@ def data_for_claims(claims)
           checked: claim.check.present?.to_s,
           checked_at: claim.check.present? ? claim.check.created_at : placeholder_date_for_nil_value,
           check_result: claim.check.present? ? claim.check.result : "",
+          number_of_days_to_check: claim.check&.number_of_days_to_check,
```

Comment:
Should this be behind a `claim.check.present?`, or `claim&.check.number_of_days_to_check`?

Reply:
Nah, we only need the safe navigation operator on the the `check`, as it could potentially be nil